sdkproject/Assets/Mapbox/Unity/Location/EditorLocationProvider.cs

```diff
@@ -21,23 +22,38 @@ public class EditorLocationProvider : AbstractEditorLocationProvider
 		string _latitudeLongitude;
 
 		/// <summary>
-		/// The mock heading value.
+		/// The transform that will be queried for location and heading data & ADDED to the mock latitude/longitude
+		/// Can be changed at runtime to simulate moving within the map.
 		/// </summary>
 		[SerializeField]
-		[Range(0, 359)]
-		float _heading;
+		Transform _targetTransform;
+
+		[SerializeField]
+		private AbstractMap _map;
+
+		private bool mapInitialized = false;
```

Comment:
Where is this ever changed to true?

Reply:
Edit: I know what it's for, but it's never set accordingly.

Reply:
You're right. I selectively committed hunks to create this PR, so it is set in my version, but not in this commit.
If you're going to create a new PR anyway can I leave it to you to fix?

Reply:
No problem!